libcxx/utils/libcxx/test/features.py

```diff
@@ -38,6 +38,39 @@ def _getAndroidDeviceApi(cfg):
         )
     )
 
+
+def _windowsSupportsModules(cfg):
```

Comment:
This makes it clear that this is only for mingw. IMO the detection logic is a bit tricky, so when clang-cl adds support I prefer to have that in a separate test.
```suggestion
def _mingwSupportsModules(cfg):
```

Reply:
Ok, sure - I applied such a rename on top.

(My intent was that since we run this for any case with `"_WIN32" in compilerMacros()`, it should be more generic, and we could branch out msvc specific detection here as well. But you're right that we can add a separate `or _msvcSupportsModules()`, in that case, which probably keeps the code easier to understand.)